README.md

```diff
@@ -1,7 +1,47 @@
 # MsgPack.Light
-MsgPack.Light is a lightweight http://msgpack.org/ format implementation.
+MsgPack.Light is a lightweight [MsgPack](http://msgpack.org/) format implementation.
 
-# Build statuses for master branch
+## Key features
+* Performance
+* .Net core compatability
+* Extensibility
+* Simple usage
+
+## Install
+Simpliest way to start using MsgPack.Light is to install [NuGet package](https://www.nuget.org/packages/MsgPack.Light/). To do that, run the following command in the  Package Manager Console:
+
+```
+ PM> Install-Package MsgPack.Light 
+```
+
+## Usage
+### Serialization to bytes array:
+```C#
+var bytes = MsgPackSerializer.Serialize(value);
+```
+
+### Deserialization:
+```C#
+var value = MsgPackSerializer.Deserialize<MyType>(bytes);
+```
+
+## Performance
+* Serialization performance is comparable with msgpack.cli
+* Deserialization performance 2-3 times faster
+* MsgPack.Light works best if a data reside a memory (*_Array benchmarks).
+* Perfoming some IO operations, performance is suboptimal, but comparable with MsgPack.Cli (*_Stream benchmarks).
+* More details can be found [here](https://github.com/roman-kozachenko/MsgPack.Light/blob/master/benchmarks.md).
+
+## Credits
+* Benchmark data authors: https://github.com/aensidhe/SerializationPerformanceTest_CSharp and https://github.com/maximn/SerializationPerformanceTest_CSharp 
```

Comment:
You can omit me ;) And we borrowed benchmark data from that C++ benchmark (link is somewhere in skype)